scripts/plugins/install_plugins.py

```diff
@@ -5,6 +5,7 @@
 import os
 import git
 import yaml
+import non_existent_module
```

Comment:
Is this just to illustrate that the UI formatting works? 
Either way this should be removed before merge

Reply:
Right, that's for illustration for you folks to get an idea of how it looks like in the PR code diff.

Will remove it before merging and also plan to squash all the commits as there's a lot of commit noise in this PR.